.github/workflows/flower-swift_sync.yml

```
@@ -0,0 +1,23 @@
+name: Sync flower-swift
+
+on:
+  push:
+    branches: ['main']
+    paths: ['src/swift/**']
+  pull_request:
+    branches: ['main']
+    paths: ['src/swift/**']
```

```suggestion
on:
  push:
    branches: ['main']
    paths: ['src/swift/**']
```

Run this only when the code merges with the main branch instead of always running it.